### ISSUE TITLE
Couldn't start client SQLTools Language Server

### ISSUE THREAD

**Issue by dking12** [mtxr/vscode-sqltools#558]
**Describe the bug**
I installed the SQLTools extension and restarted my window. I receive this error saying I can't start the server. Checking VSCode output, I see the following:
_[Error - 8:04:33 PM] Starting client failed
Launching server using runtime node failed._

**Desktop (please complete the following information):**
 - SQLTools Version: v0.21.7
 - VSCode Version: 1.44.2
 - Driver Used/Version: SQLite
 - OS: Windows

**Additional context**
Node Version: v12.16.2

**Comment by mtxr** [comment]
Hey @dking12, do you still have issues?

Can you provide more logs or that's all you see?

**Comment by dking12** [comment]
Hi @mtxr , yes I'm still having issues. What I said in my original post is the only info provided in the logs. Do you know how I could get a more detailed log?

**Comment by mtxr** [comment]
> Hi @mtxr , yes I'm still having issues. What I said in my original post is the only info provided in the logs. Do you know how I could get a more detailed log?

I still cannot reproduce this. But I believe recent releases might possible fixed this issue. Can you confirm if the issue still exists? Also, but I beliece it is, can you check if the settings are set to use node runtime? See: https://vscode-sqltools.mteixeira.dev/driver/sqlite

**Comment by dking12** [comment]
@mtxr Yep, I'm still having the problem and the only thing the log says is "Launching server using runtime node failed."

**Comment by mtxr** [comment]
Do you still use v0.21.x?

**Comment by dking12** [comment]
I'm using v0.22.9
